examples/model_selection/grid_search_automatic.py

```diff
@@ -0,0 +1,51 @@
+from __future__ import print_function
+
+from sklearn import datasets
+from sklearn.cross_validation import train_test_split
+from sklearn.grid_search import AutomaticGridSearchCV
+from sklearn.metrics import classification_report
+from sklearn.svm import SVC
+
+print(__doc__)
+
+# Loading the Digits dataset
+digits = datasets.load_digits()
+
+# To apply an classifier on this data, we need to flatten the image, to
+# turn the data in a (samples, feature) matrix:
+n_samples = len(digits.images)
+X = digits.images.reshape((n_samples, -1))
+y = digits.target
+
+# Split the dataset in two equal parts
+X_train, X_test, y_train, y_test = train_test_split(
+        X, y, test_size=0.5, random_state=0)
+
+print("# Tuning hyper-parameters")
+print()
+
+clf = AutomaticGridSearchCV(SVC(), cv=5)
+clf.fit(X_train, y_train)
+
+print("Best parameters set found on development set:")
+print()
+print(clf.best_params_)
+print()
+print("Grid scores on development set:")
+print()
+for params, mean_score, scores in clf.grid_scores_:
+    print("%0.3f (+/-%0.03f) for %r"
+          % (mean_score, scores.std() * 2, params))
+print()
+
+print("Detailed classification report:")
+print()
+print("The model is trained on the full development set.")
```

Comment:
I would call it "training set" and "test set" as these are the names used throughout the project.